packages/fxa-auth-server/scripts/test-ci.sh

```diff
@@ -6,7 +6,7 @@ cd "$DIR/.."
 export NODE_ENV=dev
 export CORS_ORIGIN="http://foo,http://bar"
 
-DEFAULT_ARGS="--require esbuild-register --recursive --timeout 5000 --exit "
+DEFAULT_ARGS="--require esbuild-register --require tsconfig-paths/register --recursive --timeout 5000 --exit "
```

Comment:
With `mocha` esbuild-register doesn't seem to work, so need to add `--require tsconfig-paths/register`